src/components/dashboard/ProductRevenueChart.tsx

```
@@ -0,0 +1,172 @@
+              {formatBRL(data.value, { locale, showSymbol: true })}
+            </span>
+          </p>
+          <p className='text-sm'>
+            <span className='text-muted-foreground'>Percentual: </span>
+            <span className='font-medium'>{data.percentage.toFixed(1)}%</span>
+          </p>
+        </div>
+      )
+    }
+    return null
+  }
```

ESLint

[error] Function component is not a function declaration

(react/function-component-definition)